### PR TITLE
Update CLI usage docs page

### PR DESCRIPTION
**Pull request by llewellyn-sl**
General updates to the CLI usage page, including:
- Updating Tower mentions
- --help command guidance for granular command info
- Improved examples
- Subsections for credentials
- Updates docs links

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -210,37 +239,38 @@ $ tw launch https://github.com/nf-core/rnaseq --params-file=./custom_rnaseq_para
     https://tower.nf/user/abhinav/watch/2XDXxX0vCX8xhx
 ```
 
-> **NOTE**: Platform CLI users are bound to the same user permissions that apply in the platform UI. Launch users can launch pre-configured pipelines in the workspaces they have access to, but they cannot add or run new pipelines.
-
-In the above command:
-
-- Pipeline level parameters are defined within the `custom_rnaseq_params.yaml` file
+- Pipeline parameters are defined within the `custom_rnaseq_params.yaml` file
```
Comment (jason-seqera) [inline USAGE.md:242]:
```suggestion
- Pipeline parameters are defined within the `custom_rnaseq_params.yaml` file.
```

**Review by jason-seqera — APPROVED** [review]
@llewellyn-sl, minor comments, thanks!

File: USAGE.md [hunk 2]
```diff
@@ -198,9 +225,11 @@ $ tw launch my_rnaseq_nf_pipeline --params-file=my_rnaseq_nf_pipeline_params_2.y
     https://tower.nf/user/abhinav/watch/2XDXxX0vCX8xhx
 ```
 
-### Launching any pipeline
+See [Nextflow configuration][nextflow-config] for more information.
 
-Platform CLI can directly launch pipelines that have not been explicitly added to the Launchpad in a platform workspace by using the full pipeline repository URL:
+### Launch an unconfigured pipeline
```
Comment (mbosio85) [inline USAGE.md:230]: Is the wording "unconfigured pipeline" clear? I find it somehow confusing even though I know this means launching a run from and URL versus from a Pipeline in the platform.

I, also, don't have a better alternative to suggest

File: USAGE.md [hunk 3]
```diff
@@ -210,37 +239,38 @@ $ tw launch https://github.com/nf-core/rnaseq --params-file=./custom_rnaseq_para
...
-- A non-primary Compute Environment has been used to launch the pipeline
+> **Note**: CLI users are bound to the same user permissions that apply in the Platform UI. Launch users can launch pre-configured pipelines in the workspaces they have access to, but they cannot add or run new pipelines.
 
 ## Workspaces
 
-Workspaces provide the context in which a user operates, i.e. to launch workflow executions, define the available resources and to manage who can access/operate on those resources. Workspaces are composed of Pipelines, Runs, Actions, Datasets, Compute Environments and Credentials. Access permissions are controlled through Participants, Collaborators, and Teams.
+Run `tw workspaces -h` to view supported workspace operations.
+Run `tw workspaces add -h` to view the required and optional fields for adding your workspace.
+
+Workspaces provide the context in which a user launches workflow executions, defines the available resources, and manages who can access those resources. Workspaces contain pipelines, runs, actions, datasets, compute environments, and credentials. Access permissions are controlled with participants, collaborators, and teams.
```
Comment (mbosio85) [inline USAGE.md:253]: Is "Workspaces contain pipelines, runs, actions, datasets, compute environments, and credentials." a comprehensive list? If so, we should add "secrets" to the list

**Review by mbosio85 — COMMENTED** [review]
I've added a couple of minor comments in the text. Otherwise, LGTM :)

File: USAGE.md (hunk 2 above)
Comment (llewellyn-sl) [reply USAGE.md:230]: Going with "unsaved pipeline". That sits better with me. WDYT?

**Review by mbosio85 — APPROVED** [review]
LGTM

**Pull request merged**